step.sh

```diff
@@ -34,7 +33,14 @@ case $lint_range in
   git diff --name-only | while read filename; do run_swiftlint "${filename}" "${reporter}" "${FLAGS}"; done
   git diff --cached --name-only | while read filename; do run_swiftlint "${filename}" "${FLAGS}"; done 
   ;;
-"all") 
-  swiftlint lint --reporter "${reporter}" ${FLAGS}
+"all")
+  output="$(swiftlint lint --reporter "${reporter}" ${FLAGS})"
+  envman add --key "SWIFTLINT_REPORT" --value "${output}"
+  echo "Saved swiftlint output in SWIFTLINT_REPORT"
+  filename="swiftlint_report.${reporter}"
+  report_path="${BITRISE_DEPLOY_DIR}/${filename}"
+  echo "${output}" > $report_path
+  envman add --key "SWIFTLINT_REPORT_PATH" --value "${report_path}"
+  echo "Saved swiftlint output in file, it's path is saved in SWIFTLINT_REPORT_PATH"
```

Comment:
Does it only work if it lints all file? Nothing for the diff?

Reply:
Yes, I suspect that the diff functionality doesn't work as expected. Please have a look a the description of this PR, I put there some details on what I think is broken, I have also created 2 bug reports.